### PR TITLE
Add possibility to create node clients

### PR DESCRIPTION
**Pull request by WojciechP**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,26 +34,48 @@ Extend your application.conf to load the module and get all goodies injectable:
...
 play.modules.enabled += "com.evojam.play.elastic4s.Elastic4sModule"
 ```
 
-You may define any number of clusters. Each of them needs to have the two above
-fields. Any other fields will be passed to Elasticsearch Java driver
-(see [Transport Client docs](https://www.elastic.co/guide/en/elasticsearch/client/java-api/current/transport-client.html)).
+You may define any number of clusters. Each of them needs to have `type`
+which is set either to `"node"` or `"transport"`. This field determines
+whether an embedded client or transport client will be created.
+Also, each cluster needs to have `cluster.name` set. Transport clients
```
Comment (ith) [inline README.md]: I would add transport client link here so it's easier to navigate https://www.elastic.co/guide/en/elasticsearch/client/java-api/current/transport-client.html

File: README.md [hunk 2]
```diff
@@ -34,26 +34,48 @@ Extend your application.conf to load the module and get all goodies injectable:
...
+these shards will be lost. Moreover, the default settings allow the embedded
+client to become the master node in the cluster. This is probably not what you
+want.
+
+As a final note on connection setup, please bear in mind that Elasticsearch
+nodes have autodiscovery system and will automatically form clusters. Therefore
+running multiple node clients in a single application may lead to strange
+behavior.
 
 You may use the `indexAndTypes` node to configure names of your indices in ES.
```
Comment (ith) [inline README.md]: indices term has been dropped - use indexes

**Comment by ith** [comment]
LGTM

**Comment by duketon** [comment]
:+1:

**Comment by albsadowski** [comment]
lgtm

**Pull request merged**